client/js/lounge.js

```diff
@@ -146,6 +146,7 @@ $(function() {
 				})
 			);
 			var channels = $.map(data.networks, function(n) {
+				sidebar.find("#network-" + n.id).data("options", n.serverOptions);
```

Comment:
If I understand this change correctly, shouldn't this be an extra `data-options="{{options}}"` attribute in [the corresponding template](https://github.com/thelounge/lounge/blob/master/client/views/network.tpl#L2) instead?